extensions/git/src/commands.ts

```diff
@@ -1236,17 +1236,29 @@ export class CommandCenter {
 		await repository.ignore(uris);
 	}
 
+	@command('git.stashIncludeUntracked', { repository: true })
```

Comment:
might be better to create a `stash` helper since the code here nearly identical to that in the `git.stash` command